examples/models/transform_jitter.py

```diff
@@ -1,3 +1,12 @@
+''' This example demonstrates how to us a jitter transform on coordinate data.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.models.sources.ColumnDataSource, bokeh.models.Circle, bokeh.models.Jitter
+    :refs: :ref:`ug_interaction` > :ref:`ug_interaction_widgets` > :ref:`ug_topics` > :ref:`ug_topics_categorical_scatters_jitter`
```

Comment:
Ah sorry I missed this earlier, we stopped trying to do the "breadcrumbs" chain of links, it's just too hard to maintain by hand. All we need are the terminal section references (seperated by commas)
```suggestion
    :refs: :ref:`ug_interaction_widgets`, :ref:`ug_topics_categorical_scatters_jitter`
```